android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -36,9 +38,11 @@
 
 import java.util.EnumSet;
 
-public class RNRadarModule extends ReactContextBaseJavaModule {
+public class RNRadarModule extends ReactContextBaseJavaModule implements PermissionListener {
 
     private static final String TAG = "RNRadarModule";
+    private static final int PERMISSIONS_REQUEST_CODE = 20160525;
```

Comment:
where does this constant come from?

Reply:
Just a random int that we use for `requestCode`. `2016-05-25` is the date Radar was incorporated, so we use it all over the place (e.g., https://github.com/radarlabs/radar-sdk-android/blob/abbdda744e0de4d44c71f7f1612c2d0cb5999d05/sdk/src/main/java/io/radar/sdk/RadarJobScheduler.kt#L27)

Reply:
gotcha, yeah let's just add a comment above it, something like:

```
// random request code (Radar's birthday!)
```

So people know where it comes from